ui/controls.js

```diff
@@ -414,10 +414,10 @@ shaka.ui.Controls = class extends shaka.util.FakeEventTarget {
 
   /**
    * @export
-   * @return {shaka.cast.CastProxy}
+   * @return {shaka.ui.CastProxyContainer}
    */
-  getCastProxy() {
-    return this.castProxy_;
+  getCastProxyContainer() {
+    return this.castProxyContainer_;
```

Comment:
Instead of changing an exported method name and type (a breaking change), I would suggest you have a stubbed version of CastProxy itself instead, with the same name and interface.

Reply:
Good idea.
It also ended up being shorter and simpler than defining two classes, so that's a plus.